application/app/services/dataverse/user_service.rb

```diff
@@ -0,0 +1,29 @@
+module Dataverse
+  class UserService
+    include LoggingCommon
+    include DateTimeCommon
+
+    AUTH_HEADER = 'X-Dataverse-key'
+    class UnauthorizedException < Exception; end
+    class ApiKeyRequiredException < Exception; end
+
+    def initialize(dataverse_url, http_client: Common::HttpClient.new(base_url: dataverse_url), api_key: nil, file_utils: Common::FileUtils.new)
```

Comment:
Do we need file_utils: Common::FileUtils.new?

Reply:
no, let me remove it